origin-growth/src/rules/rules.js

```diff
@@ -123,6 +169,57 @@ class Campaign {
     }
     return rewards
   }
+
+  /**
```

Comment:
These methods are not really related to the rules calculation.
So maybe we should put them under src/resources/campaign.js ?
We can discuss. This does not have to be done in this PR.

Reply:
Cool have added an item to [our document](https://docs.google.com/document/d/15nzudW56hQVIWHc7DRJI5XzxenM0-oc2p7z4vOqc8ms/edit?ts=5b7b483f#heading=h.b3i2k22a9u94)